language/move-compiler/src/diagnostics/mod.rs

```diff
@@ -145,7 +145,7 @@ fn convert_loc(file_mapping: &FileMapping, loc: Loc) -> (FileId, Range<usize>) {
     (id, range)
 }
 
-fn render_diagnostic(
+pub fn render_diagnostic(
```

Comment:
Why was this made public?

Reply:
I was using it in the language server before I discovered `into_codespan_format` and forgot to change it back. Fixed now!